app/src/main/java/org/mozilla/focus/utils/StorageUtils.java

```diff
@@ -91,8 +91,17 @@ static File getFirstRemovableMedia(@NonNull Context ctx) {
         final File[] files = ctx.getExternalMediaDirs();
         for (final File file : files) {
             // on some devices such as Oppo, it might return null
-            if ((file != null) && Environment.isExternalStorageRemovable(file)) {
-                return file;
+            if ((file != null)) {
```

Comment:
Would it be simpler to just do:

try {
   if ((file != null) && Environment.isExternalStorageRemovable(file)) {
   }
}